src/solve.jl

```diff
@@ -171,23 +239,179 @@ function solve!(integrator::DDEIntegrator)
     # clean up solution
     postamble!(integrator)
 
-    # calculate and add errors to solution if analytic solution to problem exists
-    # can not use same mechanism as for ODE problems since analytic solutions for DDE depend
-    # on initial value u0
+    if integrator.opts.save_everystep && isempty(integrator.opts.saveat)
```

Comment:
Can this be pulled out to a function call on the integrator? It would make `solve` a bit more legible.